apps/medusa-be/.env.template

```
@@ -39,3 +39,11 @@ SETTINGS_ENCRYPTION_KEY=
 # Credentials and settings are managed in admin: Settings → PPL
 FEATURE_PPL_ENABLED=0
 PPL_ENVIRONMENT=testing
+
+# Company Check Module
+# Set to "1" to enable Company Check module, "0" to disable
+FEATURE_COMPANY_ENABLED=0
+# Base URL for VIES REST API (company-check Vies client)
+VIES_BASE_URL=https://ec.europa.eu/taxation_customs/vies/rest-api
+# WSDL URL for Moje Dane SOAP API (company-check)
+MOJE_DANE_WSDL_URL=https://adisrws.mfcr.cz/dpr/axis2/services/rozhraniCRPDPH.rozhraniCRPDPHSOAP
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MOJE_DANE_WSDL_URL key should go before the VIES_BASE_URL key

(UnorderedKey)